protos/configs.proto

```diff
@@ -515,6 +515,32 @@ message ActionConfig {
     // TODO(ekrekr): add a notebook runtime field definition.
   }
 
+  message DataPreparationConfig {
+    // The name of the data preparation.
+    string name = 1;
+
+    // Targets of actions that this action is dependent on.
+    repeated Target dependency_targets = 2;
```

Comment:
You can just call it `dependencies` as the type is already Target, no need to repeat it in the field name.